### PR TITLE
Release 0.2.41

### PR DESCRIPTION
**Pull request by johnc1231**
Stacked on #8799, releasing for a somewhat significant bug fix.

### CODE REVIEW

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.41
+
+Release 2020-05-15
+
+### Bug fixes
+
+- (hail#8799, hail#8786) Fixes ArrayOutOfBoundsException seen in pipelines that reuse a tuple value.
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md:31]: ArrayIndexOutOfBoundsException (minor gripe, but perhaps better searchability)

File: hail/python/hail/docs/change_log.md [hunk 2]
```diff
@@ -22,6 +22,16 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.41
+
+Release 2020-05-15
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md:27]: missing a `d`.

**Comment by tpoterba** [comment]
thanks for leading the charge to fix this stuff fast!

**Comment by johnc1231** [comment]
Also added one more entry since one of nawatts's PRs went in.

**Pull request merged**